### PR TITLE
Update docstring for weights_only

### PR DESCRIPTION
**Pull request by mikaylagawarecki**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #127575

Differential Revision: [D58023165](https://our.internmc.facebook.com/intern/diff/D58023165)

### CODE REVIEW

**Review by malfet — APPROVED** [review]
LGTM

File: torch/serialization.py [hunk 1]
```diff
@@ -921,7 +921,8 @@ def load(
         pickle_module: module used for unpickling metadata and objects (has to
             match the :attr:`pickle_module` used to serialize file)
         weights_only: Indicates whether unpickler should be restricted to
-            loading only tensors, primitive types and dictionaries
+            loading only tensors, tensor subclasses, primitive types, dictionaries
+            and any types in :func:`torch.serialization.get_safe_globals`.
```
Comment (malfet) [inline torch/serialization.py:925]: Not sure if pointing to get_safe_globals is sufficient, as main API is `add_safe_globals` isn't it?
```suggestion
            and any types added via  :func:`torch.serialization.add_safe_globals`.
```

**Comment by mikaylagawarecki** [comment]
@mikaylagawarecki has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D58023165).

**Pull request closed without merging**